Cannot Use in Other Bazel Project

Hi all,

Although this project builds well with Bazel, we are having difficulty using it in another Bazel project (https://github.com/prysmaticlabs/prysm) as a dependency. 

The failure is as follows:
```
ERROR: /home/nishant/.cache/bazel/_bazel_nishant/2d55d16d3d0901116c0ae97899c3d124/external/com_github_prysmaticlabs_go_bls/BUILD.bazel:3:1: C++ compilation of rule '@com_github_prysmaticlabs_go_bls//:go_default_library%linux_amd64%cgo_c_lib' failed (Exit 1) gcc failed: error executing command /usr/bin/gcc -U_FORTIFY_SOURCE -fstack-protector -Wall -Wunused-but-set-parameter -Wno-free-nonheap-object -fno-omit-frame-pointer -MD -MF ... (remaining 66 argument(s) skipped)

crypto.go:5:10: fatal error: external/herumi_mcl/include/mcl/bls.h: No such file or directory
compilation terminated.
```

Even after adding the transitive deps the Prysm WORKSPACE:
```
go_repository(
    name = "com_github_prysmaticlabs_go_bls",
    commit = "a37e65522727d6a8518151ce9008aa155632c13d",
    importpath = "github.com/prysmaticlabs/go-bls",
)

load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")

git_repository(
    name = "herumi_mcl",
    commit = "79b3a33e21072712f00985ed2adf34b3bcf0d74e",
    remote = "https://github.com/prysmaticlabs/mcl",
)

git_repository(
    name = "bazelify_gmp",
    commit = "bb4881b35e6864c90493980d035e1d984cafd093",
    remote = "https://github.com/robin-thomas/bazelify-gmp",
)

git_repository(
    name = "boringssl",
    commit = "fafc4482e85c09e7af5f71b2eb287b73ccd1020a",
    remote = "https://github.com/google/boringssl",
)

git_repository(
    name = "io_bazel_rules_m4",
    commit = "2bf69df77dfb6b3ba6b7fc95c304b0dc279375bc",
    remote = "https://github.com/jmillikin/rules_m4",
)

load("@io_bazel_rules_m4//:m4.bzl", "m4_register_toolchains")

m4_register_toolchains()
```

Any help on this would be much appreciated.

--- a/bls.go
+++ b/bls.go
@@ -3,7 +3,7 @@ package bls
 /*
 #cgo CFLAGS:-DMCLBN_FP_UNIT_SIZE=6 -DMCL_DONT_USE_OPENSSL -I ${SRCDIR}/mcl/include/mcl -I ${SRCDIR}/mcl/lib
 #cgo LDFLAGS:-L${SRCDIR}/mcl/lib -lbls384_dy -lstdc++
-#include "external/herumi_mcl/include/mcl/bls.h"
+#include "include/mcl/bls.h"
 */
 import "C"
 import "fmt"

--- a/crypto.go
+++ b/crypto.go
@@ -2,7 +2,7 @@ package bls
 
 /*
 #cgo CFLAGS:-DMCLBN_FP_UNIT_SIZE=6
-#include "external/herumi_mcl/include/mcl/bls.h"
+#include "include/mcl/bls.h"
 */
 import "C"
 import "fmt"

